### ISSUE TITLE
add `grid-styled`

### ISSUE THREAD

**Issue by mar-bi** [chingu-voyage3/grad.then#6]
https://github.com/jxnblk/grid-styled

**Comment by mar-bi** [comment]
Do we really need this plug-in?  :confused: 
It seems to me that we can use pure `CSS Grid` without a framework wrapping. 
The same like we did with a typography - put it into the `global style`.
Are there other arguments for using it?  :smiley:

**Comment by spences10** [comment]
If you're confident we can achieve what we need to do without it then I'm with you @mar-bi

**Comment by mar-bi** [comment]
Thank you, @spences10  :blush:  :smiley:

@miljan-fsd suggested to use it. 
I went through the docs for this package twice. It has some built-in grid with 3 breakpoints. So it would be like using **Bootstrap**, but the initial idea was to avoid packages with grid system **out of the box** and learn how to use **CSS Grid** on its own.

So, I decided to dive into **CSS Grid** and built a `12-cols grid` for the `main page` and `reusable components`. Then we can modify it, add breakpoints, and use extensively.  :laughing:
